src/main/groovy/org/pih/warehouse/api/PaginationParams.groovy

```diff
@@ -2,7 +2,7 @@ package org.pih.warehouse.api
 
 class PaginationParams {
 
-    Integer offset = 0
+    Integer offset
 
-    Integer max = 10
+    Integer max
```

Comment:
Probably I missed some part of the discussion about it, but what was the reason for deleting the default value of those fields?

Reply:
oh, right